literature-cleaned.bib

```diff
@@ -10411,6 +10501,15 @@ @article{ALMK:SCP10
 	year = 2010
 }
 
+@article{MN:CEP2010,
+	title = {{Mann-Whitney U Test}},
+	author = {McKnight, Patrick E and Najab, Julius},
+	journal = {The Corsini Encyclopedia of Psychology},

```

Comment:
s.a.

Reply:
Could not find an official abbreviation.